homeassistant/components/varta_storage/__init__.py

```diff
@@ -0,0 +1,252 @@
+            "native_value": result.error_code,
+            "device_class": "None",
+            "state_class": "None",
+            "native_unit_of_measurement": None,
+        }
+        thisdict[11] = {
+            "name": "VARTA total charged energy",
+            "native_value": result.total_charged_energy,
+            "device_class": "energy",
+            "state_class": "total",
+            "native_unit_of_measurement": ENERGY_KILO_WATT_HOUR,
+        }
+        thisdict[12] = {
+            "name": "VARTA total grid to home",
+            "native_value": result.grid_to_home,
+            "device_class": "energy",
+            "state_class": "total",
+            "native_unit_of_measurement": ENERGY_WATT_HOUR,
+        }
+        thisdict[13] = {
+            "name": "VARTA total home to grid",
+            "native_value": result.home_to_grid,
+            "device_class": "energy",
+            "state_class": "total",
+            "native_unit_of_measurement": ENERGY_WATT_HOUR,
+        }
+        thisdict[14] = {
+            "name": "VARTA invertert total charged",
+            "native_value": result.inverter_total_charged,
+            "device_class": "energy",
+            "state_class": "total",
+            "native_unit_of_measurement": ENERGY_WATT_HOUR,
+        }
+        thisdict[15] = {
+            "name": "VARTA invertert total discharged",
+            "native_value": result.inverter_total_discharged,
+            "device_class": "energy",
+            "state_class": "total",
+            "native_unit_of_measurement": ENERGY_WATT_HOUR,
+        }
+        thisdict[16] = {
+            "name": "VARTA charge cycle counter",
+            "native_value": result.charge_cycle_counter,
+            "device_class": "None",
+            "state_class": "None",
+            "native_unit_of_measurement": None,
+        }
+        thisdict[17] = {
+            "name": "VARTA hours until filter maintenance",
+            "native_value": result.hours_until_filter_maintenance,
+            "device_class": "None",
+            "state_class": "None",
+            "native_unit_of_measurement": TIME_HOURS,
+        }
+        thisdict[18] = {
+            "name": "VARTA fan",
+            "native_value": result.fan,
+            "device_class": "None",
+            "state_class": "None",
+            "native_unit_of_measurement": None,
+        }
+        thisdict[19] = {
+            "name": "VARTA main",
+            "native_value": result.main,
+            "device_class": "None",
+            "state_class": "None",
+            "native_unit_of_measurement": None,
+        }
+
+        return thisdict
+
+    coordinator = DataUpdateCoordinator(
+        hass,
+        _LOGGER,
+        # Name of the data. For logging purposes.
+        name="sensor",
+        update_method=async_update_data,
+        # Polling interval. Will only be polled if there are subscribers.
+        update_interval=timedelta(seconds=1),
```

Comment:
We should not be polling every second, let's make this more reasonable in terms of what would be a good balance.